cypress/e2e/aboutpage_spec.cy.js

```diff
@@ -0,0 +1,32 @@
+describe('about page spec', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://lucifer-quotes.vercel.app/api/quotes/10', {
+      statusCode: 200,
+      fixture: 'quotes'
+    })
+    cy.visit('http://localhost:3000/About')
+  });
+
+  it('Should display the navbar with a logo the takes user back to homepage', () => {
+    cy.get('nav').find('.about-container').contains('h2', 'About')
+      .get('nav').find('.logo-container').contains('h1', 'LuciVerse')
+      .click()
+      .url().should('eq', 'http://localhost:3000/')
+  })
+
+  it('Should display title and Tv show description', () => {
+    cy.get('.about-header').contains('About The Show')
```

Comment:
Instead of using .get() and .contains() separately for each element, you can combine them into a single assertion by using the 'contains' assertion directly on the element. For example, you can use .contains('.about-header', 'About The Show') to assert that the about header contains the expected text!